src/engine/DrawManager.java

```diff
@@ -1019,17 +1019,14 @@ private Color slowlyChangingColors(String color) {
 			if (c == '8') return new Color(55, 55, 55);
 			if (c == '9') return new Color(65, 65, 65);
 		}
-		if (color == "RAINBOW") {
-			if (c == '0') return new Color(254, 254, 0);
-			if (c == '1') return new Color(135, 254, 0);
-			if (c == '2') return new Color(0, 254, 0);
-			if (c == '3') return new Color(0, 254, 254);
-			if (c == '4') return new Color(0, 135, 254);
-			if (c == '5') return new Color(0, 0, 254);
-			if (c == '6') return new Color(135, 0, 205);
-			if (c == '7') return new Color(254, 0, 224);
-			if (c == '8') return new Color(254, 0, 135);
-			if (c == '9') return new Color(220, 200, 254);
+		if (color == "RANDOM") {
```

Comment:
[P1]

Through my own code review, 
I found that the feature implemented in the blinkingColor method works similarly to when ```color == "HIGH_SCORES"```.

Accordingly, I deleted the existing color algorithm and proceeded with further development using the slowlyChangingColors method.

Thank you.

Reply:
Added comments to test code. Thank you for your feedback.